lib/models/base-locale.js

```diff
@@ -26,18 +26,22 @@ const emailsValidation = Joi.array().min(1).items(Joi.string().email())
 
 const createSchema = Joi.object().keys({
   nom: nomValidation.required(),
-  emails: emailsValidation.required()
+  emails: emailsValidation.required(),
+  commune: Joi.string()
+    .valid(...getCodesCommunes())
+    .required()
+    .messages({'any.required': '"commune" is required', 'any.only': 'Code commune inconnu'})
```

Comment:
Il n'est pas nécessaire de spécifier `'"commune" is required'` car c'est le message par défaut de `Joi`. En revanche on pourrait le traduire mais cette PR a déjà trop souffert ;) On verra ça une autre fois.

```suggestion
    .messages({'any.only': 'Code commune inconnu'})
```